src/main/java/com/marklogic/appdeployer/command/temporal/DeployTemporalCollectionsCommand.java

```diff
@@ -0,0 +1,36 @@
+package com.marklogic.appdeployer.command.temporal;
+
+import com.marklogic.appdeployer.command.AbstractResourceCommand;
+import com.marklogic.appdeployer.command.CommandContext;
+import com.marklogic.appdeployer.command.SortOrderConstants;
+import com.marklogic.mgmt.ResourceManager;
+import com.marklogic.mgmt.temporal.TemporalCollectionManager;
+
+import java.io.File;
+
+public class DeployTemporalCollectionsCommand extends AbstractResourceCommand {
+
+	private String databaseIdOrName;
+
+	public DeployTemporalCollectionsCommand() {
+		setExecuteSortOrder(SortOrderConstants.DEPLOY_TEMPORAL_COLLECTIONS);
+		// if the temporal collection contains documents, then the delete operation will fail
+		// TODO - could add check to get count of documents in temporal collection. If zero docs, then can delete
```

Comment:
I think this is fine too. "Undeploying" temporal config can be a future enhancement.